mandatory/4-tax.js

```diff
@@ -5,8 +5,11 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
-
+function calculateSalesTax(num1) {
+   return (num1 * 20)/ 100 + num1;
+}
+let total = calculateSalesTax(15);
```

Comment:
try to remove unused codes. Can you check why the tests are failing?

Reply:
This test passed on my machine Amalia.

Reply:
Sorry, I just checked. The tests did not pass for 3-function-output.js